rasa/cli/train.py

```diff
@@ -178,41 +177,3 @@ def extract_nlu_additional_arguments(args: argparse.Namespace) -> Dict:
         arguments["num_threads"] = args.num_threads
 
     return arguments
-
-
-def _get_valid_config(
-    config: Optional[Text],
-    mandatory_keys: List[Text],
-    default_config: Text = DEFAULT_CONFIG_PATH,
-) -> Text:
-    """Get a config from a config file and check if it is valid.
-
-    Exit if the config isn't valid.
```

Comment:
Are these changes needed?

Reply:
Not sure if they are related to the changes of this PR.

Reply:
I've moved this function to a utils package because I am using it inside the train/test loop (from `cli/data.py`). Wdyt - is it fine to keep it as a public fn in a utils pkg?

Reply:
We should ask @federicotdn about this.

Reply:
That sounds good :+1: